tests/app/test_jinja_filters.py

```diff
@@ -153,9 +153,6 @@ def test_format_percentage(percentage, formatted_percentage):
         ("duration-year", 100, "short", "100 bl", "cy"),
         ("duration-hour", 100, "long", "100 awr", "cy"),
         ("duration-year", 100, "long", "100 mlynedd", "cy"),
-        ("mass-metric-ton", 100, "long", "100 tonnes", "en_GB"),
-        ("mass-metric-ton", 1, "long", "1 tonne", "en_GB"),
-        ("mass-metric-ton", 100, "short", "100 t", "en_GB"),
```

Comment:
I think you've deleted this by accident

Reply:
Hmm weird, I didn't even touch that myself....